app/javascript/components/collections/CollectionsRepoList.vue

```diff
@@ -119,13 +127,21 @@
 </template>
 <script setup>
   import { computed, ref, onMounted, inject } from 'vue'
-
   import RepoItem from '../shared/RepoItem.vue'
   import ApplicationSpaceItem from '../application_spaces/ApplicationSpaceItem.vue'
+  import jwtFetch from '../../packs/jwtFetch'
```

Comment:
jwtFetch has been refactored, please merge latest code and update it accordingly

Reply:
down